### ISSUE TITLE
Pagination doesn't work the same here as in JSON API

### ISSUE THREAD

**Issue by aickin** [CityOfLosAngeles/mobility-data-specification#49]
In the provider API definition, it says that "pagination must comply with the JSON API specification", but the JSON API spec says:

> Pagination links MUST appear in the `links` object that corresponds to a collection. To paginate the primary data, supply pagination links in the top-level `links` object.

This spec puts the `first`, `last`, `prev`, `next` URLs at the top level of the response, rather than inside in a `links` object. I feel like either this spec should use the `links` object or not say that paginations complies with JSON API.

**Comment by thekaveman** [comment]
@aickin good catch, that was just me being lazy and not reading that spec close enough. Happy to take a PR to fix this if you'd like.

Also see our discussion on #46 where this certainly matters. There will be an incoming PR for exploring that idea more, shortly.

**Comment by thekaveman** [comment]
@aickin can you take a look at the PR and see if this makes sense with your understanding?
